### PR TITLE
docs(Introduction): Add contact and community entires to Intro section 

### LINKED ISSUE #2029: Add "Community Extensions" and "Get Help" sections to Zeebe docs

### PR DESCRIPTION
**Pull request by wints**
I reworked the Intro section of the docs so that it's more complete for a new user:
• Added a "Community Contributions" page
• Added a "Get Help & Get Involved" page
• Removed the "First Contact" page from the left-side nav and moved this information to the README instead

closes #2029

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3d144f7a8de405832bb03c3ab656c2eb6839712f
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

File: docs/src/introduction/community-contributions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+We use [Awesome Zeebe](https://awesome.zeebe.io/) as a place to keep track of Zeebe...
+
+* Clients
+* Workers
+* Exporters
+* Applications
+
+...along with other integrations such as Spring-Zeebe and an Apache Kafka connector.
+
+If you contributed something to Zeebe, we encourage you to [add it to Awesome Zeebe via pull request](https://github.com/zeebe-io/awesome-zeebe/blob/master/CONTRIBUTING.md).
```
Comment (npepinpe) [inline docs/src/introduction/community-contributions.md]: I think the distinction between contributing to Zeebe and contributing to the Zeebe ecosystem is not clear here.

File: docs/src/introduction/get-help-get-involved.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Get Help and Get Involved
+
+To support the open-source community, we provide public-facing Zeebe support channels and a way for users to report problems and make contributions.
```
Comment (npepinpe) [inline docs/src/introduction/get-help-get-involved.md]: I would remove the second support (`Zeebe support channels`) to avoid repeating it

**Review by npepinpe — APPROVED** [review]
Looks good in general, I added two comments but I wouldn't consider them blocker.

File: docs/src/introduction/community-contributions.md (hunk 1 above)
Comment (wints) [reply docs/src/introduction/community-contributions.md]: Good point, I updated this.

File: docs/src/introduction/get-help-get-involved.md (hunk 2 above)
Comment (wints) [reply docs/src/introduction/get-help-get-involved.md]: Agree, updated.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3d144f7a8de405832bb03c3ab656c2eb6839712f
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

**Pull request merged**